views/content/crowdsale.md

```diff
@@ -26,68 +26,83 @@ Also, generally those who are funding can't have any say on how the money is spe
 
 Now copy this code and let's create the crowdsale:
 
-     
-    contract token { function transfer(address receiver, uint amount){  } }
-    
-
-    contract Crowdsale {
-        address public beneficiary;
-        uint public fundingGoal; uint public amountRaised; uint public deadline; uint public price;
-        token public tokenReward;   
-        Funder[] public funders;
-        event FundTransfer(address backer, uint amount, bool isContribution);
-        bool crowdsaleClosed = false;
-        
-        /* data structure to hold information about campaign contributors */
-        struct Funder {
-            address addr;
-            uint amount;
-        }
-        
-        /*  at initialization, setup the owner */
-        function Crowdsale(
-            address ifSuccessfulSendTo, 
-            uint fundingGoalInEthers, 
-            uint durationInMinutes, 
-            uint etherCostOfEachToken, 
-            token addressOfTokenUsedAsReward
-        ) {
-            beneficiary = ifSuccessfulSendTo;
-            fundingGoal = fundingGoalInEthers * 1 ether;
-            deadline = now + durationInMinutes * 1 minutes;
-            price = etherCostOfEachToken * 1 ether;
-            tokenReward = token(addressOfTokenUsedAsReward);
-        }   
-        
-        /* The function without name is the default function that is called whenever anyone sends funds to a contract */
-        function () {
-            if (crowdsaleClosed) throw;
-            uint amount = msg.value;
-            funders[funders.length++] = Funder({addr: msg.sender, amount: amount});
-            amountRaised += amount;
-            tokenReward.transfer(msg.sender, amount / price);
-            FundTransfer(msg.sender, amount, true);
-        }
-            
-        modifier afterDeadline() { if (now >= deadline) _ }
-
-        /* checks if the goal or time limit has been reached and ends the campaign */
-        function checkGoalReached() afterDeadline {
-            if (amountRaised >= fundingGoal){
-                beneficiary.send(amountRaised);
-                FundTransfer(beneficiary, amountRaised, false);
-            } else {
-                for (uint i = 0; i < funders.length; ++i) {
-                  funders[i].addr.send(funders[i].amount);  
-                  FundTransfer(funders[i].addr, funders[i].amount, false);
-                }               
+contract token { function transfer(address receiver, uint amount){  } }
```

Comment:
Changes starting from here break the code formatting of the example and make the diff hard to read